Angular/package.json

```diff
@@ -42,14 +42,14 @@
     "eslint-plugin-import": "2.22.1",
     "eslint-plugin-jsdoc": "30.7.6",
     "eslint-plugin-prefer-arrow": "1.2.2",
-    "jasmine-core": "~3.6.0",
+    "jasmine-core": "~3.8.0",
     "jasmine-spec-reporter": "~5.0.0",
-    "karma": "~6.1.0",
+    "karma": "~6.4.0",
     "karma-chrome-launcher": "~3.1.0",
     "karma-coverage": "~2.0.3",
     "karma-jasmine": "~4.0.0",
     "karma-jasmine-html-reporter": "^1.5.0",
```

Comment:
"karma-jasmine-html-reporter": "^1.7.0",
Can't we also karma-jasmine-html-report to the latest stable version ?
<img width="581" alt="image" src="https://user-images.githubusercontent.com/47911842/177130998-85e53064-a54b-441e-9cc7-75cd2f8ad902.png">